### PR TITLE
Add mongoDB Connection to Locationpage

### LINKED ISSUE #32: Connect "Location"-Page to Database
- [ ] #2 
- [ ] #10 
- [ ] Connect Page to database

### PR DESCRIPTION
**Pull request by lauravikanis**
Add a databank connection to Location.js so ist can display one specific location
close #32

### CODE REVIEW

File: client/src/components/Card/LocationDetails.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  width: 100%;
+  min-height: 50px;
+  margin: 15px auto;
+  padding: 0 2.5rem;
+  visibility: ${(props) => (props.hidden ? "hidden" : "visible")};
+
+  p {
+    color: var(--secondary-color);
+    text-align: center;
+    align-self: center;
```
Comment (mariegross) [inline client/src/components/Card/LocationDetails.js:20]: Do you really need both when it's referring to a `p` element?

**Review by mariegross — APPROVED** [review]
Just left one little suggestion - looks great otherwise! Congratulations on mastering mongoDB 🥳

File: client/src/components/Card/Card.stories.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ Location.args = {
   count: false,
   children: (
     <p>
-      Name <br /> Adress, <br />
+      Name <br /> Address, <br />
```
Comment (phlgr) [inline client/src/components/Card/Card.stories.js:40]: Please don't use br. This is a visual component thing and should not be hard-coded

File: client/src/pages/Location.js [hunk 3]
```diff
@@ -2,34 +2,58 @@ import React from "react";
...
+import mappath from "../assets/map.svg";
 
-const TourDiv = styled.div`
+import { Link, useParams } from "react-router-dom";
+import { useQuery } from "react-query";
+import { getLocationByName } from "../api/locations";
+import DetailCard from "../components/Card/LocationDetails";
+import PictureContainer from "../components/Image/Picture";
+
+const LocationDiv = styled.div`
```
Comment (phlgr) [inline client/src/pages/Location.js:14]:
```suggestion
const LocationContainer = styled.div`
```

Comment (lauravikanis) [reply client/src/pages/Location.js:14]: Every page container is named consistently. Such as "HomeDiv", "TourDiv"," LocationDiv". I don't think I should change this

**Pull request merged**